packages/website/styles/layouts/pageContainer.scss

```diff
@@ -1,7 +1,8 @@
 .page-container {
-  height: 100vh; // TODO: Base off 100vh - header/footer heights
+  // height: 100vh; // TODO: Base off 100vh - header/footer heights
```

Comment:
@MichaelPhan I tweaked the `.corkscrew-background` class a bit so that the image would fill the page but not overflow past the footer. Commenting out this line was necessary for my approach but I left it because I'm not sure if you had a plan for this?